tuxemon/states/choice/choice_monster.py

```diff
@@ -0,0 +1,112 @@
+# SPDX-License-Identifier: GPL-3.0
+# Copyright (c) 2014-2023 William Edwards <shadowapex@gmail.com>, Benjamin Bean <superman2k5@gmail.com>
+from __future__ import annotations
+
+import math
+from collections.abc import Callable, Sequence
+from functools import partial
+from typing import Any
+
+import pygame_menu
+from pygame_menu.locals import ALIGN_CENTER, POSITION_CENTER, POSITION_EAST
+from pygame_menu.widgets.selection.highlight import HighlightSelection
+
+from tuxemon import prepare
+from tuxemon.animation import Animation
+from tuxemon.db import MonsterModel, db
+from tuxemon.menu.menu import PygameMenuState
+from tuxemon.menu.theme import get_theme
+from tuxemon.tools import transform_resource_filename
+
+ChoiceMenuGameObj = Callable[[], None]
+
+
+class ChoiceMonster(PygameMenuState):
+    """
+    Game state with a graphic box and some text in it.
+
+    Pressing the action button:
+    * if text is being displayed, will cause text speed to go max
+    * when text is displayed completely, then will show the next message
+    * if there are no more messages, then the dialog will close
+    """
+
+    def __init__(
+        self,
+        menu: Sequence[tuple[str, str, Callable[[], None]]] = (),
+        escape_key_exits: bool = False,
+        **kwargs: Any,
+    ) -> None:
+        theme = get_theme()
+        theme.scrollarea_position = POSITION_EAST
+        theme.scrollbar_color = (237, 246, 248)
+        theme.scrollbar_slider_color = (197, 232, 234)
```

Comment:
Color values should be extracted as constant with proper name into a constants file, so we could see what color is set (because I dare anybody to guess which color `(197, 232, 234)` matches lol) and so we could reuse these colors in other places as well.

Reply:
Planned in another PR, the one regarding the style